### ISSUE TITLE
`document.adoptedStyleSheets` is frozen

### ISSUE THREAD

**Issue by macdonst** [zachleat/table-saw#1]
On this [line](https://github.com/zachleat/table-saw/blob/main/table-saw.js#L70) of the WC you use `push` to add a new sheet to the `document.adoptedStyleSheets` array. 

I understand the proper way to do this is:

```javascript
document.adoptedStyleSheets = [...document.adoptedStyleSheets, extraSheet];
```

as per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/adoptedStyleSheets#append_a_new_stylesheet).

I could be totally off base but thought I should flag it.

**Comment by zachleat** [comment]
Fixed in v1.0.2 thanks!

**Comment by zachleat** [comment]
Saw this also on https://web.dev/constructable-stylesheets/ too! Hadn’t seen an error from it though!
